helm/flowforge/templates/emqx.yaml

```diff
@@ -0,0 +1,240 @@
+{{- if and ( eq .Values.forge.broker.enabled true) ( eq .Values.forge.broker.teamBroker.enabled true ) -}}
+apiVersion: apps.emqx.io/v2beta1
```

Comment:
With [Capabilities](https://helm.sh/docs/chart_template_guide/builtin_objects/) we should check if this API exists.

Reply:
Something like

```
{{ if .Capabilities.APIVersions.Has('apps.emqx.io/v2beta1') }}
```